src/lib.rs

```diff
@@ -7,16 +7,20 @@
 //!
 //! ## Configuration
 //!
-//! By default, Rspotify uses the [`reqwest`](reqwest)
-//! asynchronous HTTP client with its default TLS, but you can customize both
-//! the HTTP client and the TLS with the following features:
+//! ### HTTP Client
 //!
-//! - `client-reqwest`, TLS available:
+//! By default, Rspotify uses the [`reqwest`] asynchronous HTTP client with its
+//! default TLS, but you can customize both the HTTP client and the TLS with the
+//! following features:
+//!
+//! - [`reqwest`](https://github.com/seanmonstar/reqwest): enabling
```

Comment:
Both of these link to the GitHub pages because I later reference ureq and that's not possible via an intra-doc reference. So for consistency both point out to github. But maybe I can change both to docs.rs instead if you prefer it that way.